src/routes/(console)/verify-email/+page.svelte

```diff
@@ -0,0 +1,136 @@
+<script lang="ts">
+    import Sidebar from '$lib/components/sidebar.svelte';
+    import Navbar from '$lib/components/navbar.svelte';
+    import SendVerificationEmailModal from '$lib/components/account/sendVerificationEmailModal.svelte';
+    import { writable } from 'svelte/store';
+    import { invalidate } from '$app/navigation';
+    import { goto } from '$app/navigation';
+    import { onMount } from 'svelte';
+    import { base } from '$app/paths';
+    import { Dependencies } from '$lib/constants';
+    import { page } from '$app/state';
+
+    let sideBarIsOpen = writable(false);
+    let showAccountMenu = writable(false);
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    let project: any = {
+        $id: 'verify-email-project',
+        region: 'us-east-1',
+        name: 'Verify Email Project'
+    };
+    let avatar = '/images/default-avatar.png';
+    let progressCard = {
+        title: 'Get started',
+        percentage: 33
+    };
+
+    let navbarProps = {
+        logo: {
+            src: '/images/appwrite-logo-light.svg',
+            alt: 'Appwrite Logo'
+        },
+        avatar: avatar,
+        organizations: [],
+        currentProject: project
+    };
+
+    let showVerificationModal = $state(!page.data.account?.emailVerification);
+
+    $effect(() => {
+        if (page.data.account?.emailVerification) {
+            checkEmailVerification();
+        }
+    });
+
+    async function checkEmailVerification() {
+        if (page.data.account?.emailVerification) {
+            await goto(`${base}/`);
+        }
+    }
+
+    onMount(() => {
+        if (!page.data.account) {
+            goto(`${base}/login`);
+            return;
+        }
+
+        // If email is already verified, redirect immediately
+        if (page.data.account?.emailVerification) {
+            checkEmailVerification();
+            return;
+        }
+
+        const interval = setInterval(async () => {
+            await invalidate(Dependencies.ACCOUNT);
+            checkEmailVerification();
+        }, 2000);
```

Comment:
Have you tried using realtime instead for this?

Maybe we can make the delay longer as a backup in case realtime doesn't work?

Reply:
doesn't work. current approach should be OK now.